db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_11_24_202124) do
```

Comment:
@ttomz12 Please double check this to make sure that this is OK considering the warning msg in the comment